modules/007-registrypackages/images/jq/werf.inc.yaml

```diff
@@ -1,9 +1,9 @@
-{{- $version := "1.6" }}
+{{- $version := .CandiVersionMap.jq.version }}
 {{- $image_version := $version | replace "." "-" }}
 ---
 image: {{ $.ModuleName }}/{{ $.ImageName }}-{{ $image_version }}
 from: {{ $.Images.BASE_SCRATCH }}
-git:
+fromCacheVersion: 20241223 # rm if update version
```

Comment:
`fromCacheVersion` will not be needed if switch from artifacts to image below

Reply:
done